argshook/dotfiles
# test

test

<details><summary>Full commit log</summary>

* fix qutebrowser config
    recent version introduced some breaking changes.
    `content.javascript.can_access_clipboard` option was changed to `content.javascript.clipboard`
* update rifle config to use $EDITOR for html files
* cleanup .zshrc from BUN config
* update lazy-lock.json
* improve ,pr-create command
    * before pushin PR check if remote branch exists
    * if it doesn't exist, create it
    * then run gh pr create


</details>
